src/timelog.cc

```diff
@@ -55,7 +55,7 @@ namespace {
       curr->append_note(in_event.note.c_str(), *context.scope);
 
     char buf[32];
-    std::sprintf(buf, "%lds", long((out_event.checkin - in_event.checkin)
+    std::snprintf(buf, 32, "%lds", long((out_event.checkin - in_event.checkin)
```

Comment:
According the [adfad](https://en.cppreference.com/w/cpp/io/c/fprintf) `snprintf` is available in C++11, which Ledger already requires (see [`CMakeLists.txt:23`](https://github.com/ledger/ledger/blob/master/CMakeLists.txt#L23))